.github/workflows/lint.yaml

```diff
@@ -1,20 +1,13 @@
-name: Lint
+name: Action lint
 
 on:
   pull_request:
 
 jobs:
-  actionlint:
-    name: actionlint
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash
+  call-action-lint-workflow:
+    runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Check workflow files
-        run: |
-          echo "::add-matcher::.github/actionlint-matcher.json"
-          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/v1.6.19/scripts/download-actionlint.bash)
-          ./actionlint -color
+      - name: Lint actions and workflows
+        uses: bakdata/ci-templates/actions/action-lint@838fc43bf3cdd82b253a21c2f98157b449c7253b
+        with:
+          ref: "${{ github.event.pull_request.base.ref }}"
```

Comment:
Is this ref correct here? This refers to the target branch of the pull request, so `main`, right? This means whenever you commit to the pull request you lint the main branch, but we want to lint the changes in the pull request. Sticking to the default `${{ github.ref_name }}` should be fine here

Reply:
i was having[ some troubles](https://github.com/bakdata/ci-templates/actions/runs/3856861640/jobs/6573586630) using the default `${{ github.ref_name }}`. and i thought i was somehow related to the fact the action was triggered by a PR and `${{ github.ref_name }}` wasnt finding a valid branch

Reply:
Interesting, I have never seen that error, but there has to be a ways to lint the correct branch